docker/consistency.dockerfile

```diff
@@ -4,7 +4,8 @@ USER root
 ARG REF=main
 RUN apt-get update && apt-get install -y time git pkg-config make git-lfs
 ENV VIRTUAL_ENV=/usr/local
```

Comment:
is virtual env still neeeded?

Reply:
Not sure, I don't think so. I will try to remove.